examples/time_frequency/time_frequency_erds.py

```diff
@@ -111,3 +109,75 @@
     fig.colorbar(axes[0].images[-1], cax=axes[-1])
     fig.suptitle("ERDS ({})".format(event))
     fig.show()
+
+##############################################################################
+# Similar to `~mne.Epochs` objects, we can also export data from
+# `~mne.time_frequency.EpochsTFR` and `~mne.time_frequency.AverageTFR` objects
+# to a :class:`Pandas DataFrame <pandas.DataFrame>`:
+
+df = tfr.to_data_frame(time_format=None)
+df.head()
+
+##############################################################################
+# This allows us to use additional plotting functions like
+# :func:`seaborn.lineplot` to easily plot confidence bands:
+
+df = tfr.to_data_frame(time_format=None, long_format=True)
+df['channel'].cat.reorder_categories(['C3', 'Cz', 'C4'], ordered=True,
+                                     inplace=True)
+
+freq_bands = {'delta': (0.5, 4),
+              'theta': (5, 7),
+              'alpha': (8, 14),
+              'beta': (15, 35)}
+
+
+def map_bands(freq):
+    for band, (low_lim, high_lim) in freq_bands.items():
+        if low_lim <= freq <= high_lim:
+            return band
+
+
+df['band'] = pd.Categorical(
+    df['freq'].map(map_bands),
+    categories=['beta', 'alpha', 'theta', 'delta'],
+    ordered=True)
```

Comment:
yes!  I knew there was a function for that but I couldn't remember its name.

Reply:
Ok, I'm doing this now: 
``` 
freq_bounds = {'_': 0,
               'delta': 3,
               'theta': 7,
               'alpha': 13,
               'beta': 35,
               'gamma': 140}

df['band'] = pd.cut(df['freq'], list(freq_bounds.values()),
                    labels=list(freq_bounds.keys())[1:])
```
wdyt?

Reply:
`list(freq_bounds.keys())` is the same as `list(freq_bounds)`, FYI.  Otherwise looks fine

Reply:
actually looking at [the rendering on CircleCI](https://27782-1301584-gh.circle-artifacts.com/0/dev/auto_examples/time_frequency/time_frequency_erds.html#sphx-glr-auto-examples-time-frequency-time-frequency-erds-py) I wonder if you got the call to `pd.cut` correct?  there's an empty row at the bottom of the plot and the top row looks really noisy.

Reply:
argh, that's what you get when "quickly finishing something friday night" ... thanks for spotting. 

---
> there's an empty row at the bottom of the plot

Putting this fixed the empty row for gamma frequencies:
`df['band'] = df['band'].cat.remove_unused_categories()`

---
> the top row looks really noisy

I think, that's because now only two freqs (2 & 3 Hz) go into this plot and the # of reps for the CI bootstrap is very low. It's a tiny bit better with 3 freqs (aka, having the boundary between delta and theta at 4 Hz instead) - but not so much. 
I _think_ that the actual call to to `pd.cut` is ok. I also tested it against the previous mapping method. Results were identical.